src/controllers/levelcontroller.js

```diff
@@ -0,0 +1,45 @@
+class LevelController extends Controller
+{
+    checkGoalReached()
+    {
+        let currentLevelId = GAME.model.config.level;
+        let currentLevel = GAME.model.levels[currentLevelId];
+        let completed = false;
+
+        if (currentLevel.type == 'money') {
```

Comment:
This will get bloated quickly if/when we add more levels. Could you add a todo that we need to look at this at some other time?